docs/api-integration.md

```diff
@@ -37,16 +34,15 @@ options := harness.TestSuite{}
 The Go `t *testing.T` and `harness.TestSuite` are provided to `test.Harness` which provides the implementation for testing.
 
 ```go
-Run: func(cmd *cobra.Command, args []string) {
-  testutils.RunTests("kudo", testToRun, options.Parallel, func(t *testing.T) {
+func TestWithKuttl(t *testing.T) {
     harness := test.Harness{
       TestSuite: options,
       T:         t,
     }
 
-    harness.Run()
-  })
-},
+    harness.Setup()
+	harness.Run()
```

Comment:
```suggestion
    harness.Setup()
    harness.RunTests()
```

`harness.Run()` calls `Setup()` and `RunTests()`.

Reply:
Ah, good point, looks like I followed not the best example, calling Run is suffcient, then.